scripts/lib/email-mirror-postfix

```diff
@@ -41,24 +41,46 @@ from __future__ import absolute_import
 from __future__ import print_function
 
 import os
+import ssl
 import sys
 
 from optparse import OptionParser
-from os.path import dirname, abspath
-
-BASE_DIR = dirname(dirname(dirname(abspath(__file__))))
-sys.path.append(BASE_DIR)
-import scripts.lib.setup_path_on_import
 
 import posix
-import requests
-import ujson
+import json
+
+try:
+    import urllib.request as urllib2 # type: ignore
+except ImportError:
+    import urllib2 as urllib2 # type: ignore
+
+try:
+    from urlparse import urljoin  # type: ignore
+except ImportError:
+    from urllib.parse import urljoin, urlencode  # type: ignore
+
+try:
+    from urllib import urlencode  # type: ignore
+except ImportError:
+    from urllib.parse import urlencode  # type: ignore
+
+try:
+    from urllib2 import HTTPError  # type: ignore
+except ImportError:
+    from urllib.error import HTTPError  # type: ignore
 
-from six.moves import urllib
-from six.moves import configparser
-from typing import Text
+try:
+    from ConfigParser import RawConfigParser # type: ignore
+except ImportError:
+    from configparser import RawConfigParser # type: ignore
```

Comment:
We can use the `six` library; that one is guaranteed to be installed in the system Python already.  And we can use `six.text_type` instead of `Text` to avoid needing typing.

Probably worth converting this to use that since it's clean up the code a lot?